docs/secure-connections/sauce-connect.md

```diff
@@ -4,8 +4,8 @@ title: Sauce Connect Proxy 4
 sidebar_label: Overview
 ---
 
-:::caution Sauce Connect Proxy 5 is released
-Please refer to [Sauce Connect Proxy 5 Overview](/secure-connections/sauce-connect-5/) for the most recent Sauce Connect Proxy documentation.
+:::caution Important Notice: Sauce Connect 4 is reaching end-of-life status.
```

Comment:
I believe it would be better to point to (migration guide)[https://docs.saucelabs.com/secure-connections/sauce-connect-5/migrating/] as it also points to installation.